autoload/SpaceVim/layers/lang/lua.vim

```diff
@@ -80,6 +81,7 @@ endfunction
 
 function! SpaceVim#layers#lang#lua#set_variable(opt) abort
   let s:lua_repl_command = get(a:opt, 'repl_command', '') 
+  let s:lua_foldmethod = get(a:opt, 'lua_foldmethod', 'manual')
```

Comment:
the `s:lua_foldmethon` should be init out of `set_variable` function, because this function maybe will not be called.

the layer option should remove `lua_` prefix.  because this is lua layer local option.

```toml
[[layers]]
   name = "lang#lua"
   foldmethod = "manual"
```